src/unk_020553DC.c

```diff
@@ -319,7 +319,7 @@ void sub_02055644 (FieldSystem * fieldSystem, int param1)
     return;
 }
 
-void sub_02055670 (FieldSystem * fieldSystem, int param1)
+void Sound_PlayMapBgm (FieldSystem * fieldSystem, int param1)
```

Comment:
```suggestion
void Sound_PlayMapBGM (FieldSystem * fieldSystem, int param1)
```

Reply:
wait commited this outta instinct but that build's gonna fail cause it isn't replaced elsewhere, my b